### PR TITLE
Fork winxp support to legacy

### PR DESCRIPTION
**Pull request by GrayHatter**

### CODE REVIEW

**Comment by tsudoko** [comment]
Why?

File: windows/main.c [hunk 1]
```diff
@@ -626,17 +554,18 @@ static void parsecmd(uint8_t *cmd, int len)
...
+    }
 
-    if(len > 6 && memcmp(cmd, "tox://", 6) == 0) {
-        cmd += 6;
-        len -= 6;
-    } else if (len > 4 && memcmp(cmd, "tox:", 4) == 0) {
-        cmd += 4;
-        len -= 4;
-    } else {
+    if(memcmp(cmd, "tox://", 6) != 0) {
```
Comment (irungentoo) [inline windows/main.c]: Why are you reverting this change?

Comment (GrayHatter) [inline windows/main.c]: @irungentoo didn't mean to, must have been a bad merge/rebase

**Comment by GrayHatter** [comment]
@irungentoo  I don't know where I went wrong, so I rebased off master, and combed thought and cherry picked the commits needed.

**Comment by GrayHatter** [comment]
still breaks xp support, but now it can be easily fixed. Anyone have xp that can test this out for me?

**Comment by LittleVulpix** [comment]
@GrayHatter, I have a windows XP SP3 system on a VM, can test for you if you want.

**Comment by GrayHatter** [comment]
thanks for all your help @LittleVulpix

**Comment by GrayHatter** [comment]
rebased off master, because I couldn't take the messy merges

**Pull request merged**